docs/pvacseq/errors.rst

```diff
@@ -0,0 +1,169 @@
+Input VCF VEP Annotation
+________________________
+
+**Input VCF does not contain a CSQ header. Please annotate the VCF with VEP before running it.**
+
+pVACseq requires the input VCF to be annotated by VEP. The provided input VCF
+doesn't contain a ``CSQ`` ``INFO`` header which indicates that it has not been
+annotated. :ref:`The Input File Preparation section <vep>` of the
+documentation provides instructions on how to annotate your VCF with VEP.
+
+**VCF doesn't contain VEP DownstreamProtein annotations. Please re-annotate the VCF with VEP and the Wildtype and Downstream plugins.**
+
+Although the input VCF was annotated with VEP, it is missing the required
+annotations provided by the VEP Downstream plugin. The input VCF will need to
+be reannotated using all of the required arguments as outline in the :ref:`Input
+File Preparation section <vep>` of the documentation.
+
+**VCF doesn't contain VEP WildtypeProtein annotations. Please re-annotate the VCF with VEP and the Wildtype and Downstream plugins.**
+
+Although the input VCF was annotated with VEP, it is missing the required
+annotations provided by the VEP Wildtype plugin. The input VCF will need to
+be reannotated using all of the required arguments as outline in the :ref:`Input
+File Preparation section <vep>` of the documentation.
+
+**Proximal Variants VCF does not contain a CSQ header. Please annotate the VCF with VEP before running it.**
+
+When running pVACseq with the ``--proximal-variants-vcf`` argument, that
+proximal variants VCF needs to be annotated by VEP. The provided proximal
+variants VCF
+doesn't contain a ``CSQ`` ``INFO`` header which indicates that it has not been
+annotated. :ref:`The Input File Preparation section <vep>` of the
+documentation provides instructions on how to annotate your VCF with VEP.
+
+**There was a mismatch between the actual wildtype amino acid sequence and the expected amino acid sequence. Did you use the same reference build version for VEP that you used for creating the VCF?**
+
+This error occurs when the reference nucleotide at a specific position is
+different than the Ensembl transcript nucleotide at the same position. This results in
+the mutant amino acid in the ``Amino_acids`` VEP annotation being different
+from the amino acid of the transcript protein sequence as predicted by the
+Wildtype plugin. The ``Amino_acids`` VEP annotation is based on the reference
+and alternate nucleotides of the variant while the ``WildtypeProtein``
+prediction is based on the Ensembl transcript nucleotide sequence.
+
+This points to a fundamental disagreement between the reference that was
+used during alignment and variant calling and the Ensembl reference. This
+mismatch cannot be resolved by pVACseq, which is why this error is fatal.
+
+Here are a few things that might resolve this error:
+
+- If a VEP build 38 cache was accidentially used with a build 37 VCF (or vice
+  versa), the correct cache needs to be downloaded and used during VEP annotation.
```

Comment:
This list item breaks parallelism with the other list items.  It would need to be phrased so as to say something like `If a VEP build 38 cache was accidentially used with a build 37 VCF (or vice
  versa), downloading the correct cache and using it during VEP annotation`.

That particular phrasing is a bit cumbersome.  Maybe something like:
`checking that the build of the VEP cache matches the alignment build and downloading the correct cache if there is a build mismatch (such as a build 38 cache with a build 37 VCF, or vice versa)`

Alternatively, all the other list items could have their verb tenses changed to match this one!